### PR TITLE
Add style

### PR DESCRIPTION
**Pull request by ciraganenicole**
In this pull request I:

- Add style to the bookstore app

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,4 +1,233 @@
...
+}
+
+button[type='submit'] {
+  background-color: #2f74ef;
+  color: white;
+  padding: 10px 20px;
+  margin-top: 30px;
+  border: none;
+  border-radius: 5px;
+}
```
Comment (VICTOR-CHUKWU) [inline src/index.css]: - [ ] Kindly note that this project is for styling, your button background colour does not match the background of the project design, you can try this hex value`#0290ff` as it matches, also this applies to every place you used a background color blue, please fix this
your design :point_down: 
Screenshot from 2022-05-06 11-23-07
zeplin design :point_down: 
Screenshot from 2022-05-06 11-25-05

File: src/index.css [hunk 2]
```diff
@@ -1,4 +1,233 @@
...
+  line-height: 120px;
+  text-align: center;
+  margin-top: 14px;
+  margin-left: 14px;
+  color: #1e51dc;
+  position: absolute;
+  z-index: 100;
+  font-weight: 700;
+  font-size: 2em;
+}
```
Comment (VICTOR-CHUKWU) [inline src/index.css]: - [ ] Your `circle-wrap` is not looking exactly like the one on the project design, kindly fix this. make it to look like this :point_down: 
circl-wrap

File: src/index.css [hunk 3]
```diff
@@ -1,4 +1,233 @@
...
+  color: #999;
+  margin-left: 20px;
+}
+
+input {
+  margin-bottom: 5px;
+  padding: 10px;
+  border: 2px solid #d1e0fb;
+  border-radius: 5px;
+}
```
Comment (VICTOR-CHUKWU) [inline src/index.css]: - [ ] Your input is not yet looking exactly as what we have in the design, kindly fix this
 your design :point_down: 
Screenshot from 2022-05-06 11-31-00

Zeplin design :point_down: 
Screenshot from 2022-05-06 11-34-59

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @ciraganenicole 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (Whoistolu) [reply src/index.css]: Now resolved 👍🏽

File: src/index.css (hunk 2 above)
Comment (Whoistolu) [reply src/index.css]: Now resolved 👍🏽

File: src/index.css (hunk 3 above)
Comment (Whoistolu) [reply src/index.css]: Now resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @ciraganenicole 

Great job implementing the recommended changes 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**